xml/System.Net.NetworkInformation/PingReply.xml

```diff
@@ -355,6 +356,7 @@
   
  :::code language="cpp" source="~/snippets/cpp/VS_Snippets_Remoting/NclPingSync/CPP/syncping.cpp" id="Snippet1":::
  :::code language="csharp" source="~/snippets/csharp/System.Net.NetworkInformation/IPStatus/Overview/syncping.cs" id="Snippet1":::  
+ :::code language="fsharp" source="~/snippets/csharp/System.Net.NetworkInformation/IPStatus/Overview/syncping.fs" id="Snippet1":::  
```

Comment:
```suggestion
 :::code language="fsharp" source="~/snippets/csharp/System.Net.NetworkInformation/IPStatus/Overview/syncping.fs" id="Snippet1":::  
```
```suggestion
 :::code language="fsharp" source="~/snippets/fsharp/System.Net.NetworkInformation/IPStatus/Overview/syncping.fs" id="Snippet1":::  
```

Reply:
Hopefully I got it right this time.

I couldn't find a way to check this locally before submitting, is there one?  I wrote [a script to verify snippets](https://github.com/pyjarrett/MyDotNetDocTools/blob/main/CheckSnippets/Program.fs) to check this locally.

Reply:
There may be some functionality in the docs authoring VS Code plugin (can't find the link now). But Usually, I just check build errors on the PR.